tests/test_agrregation_region_aggrfuncs.py

```diff
@@ -0,0 +1,241 @@
+import pyam
+import pandas as pd
+import numpy as np
+import math
+import pytest
+
+
+all_regs = ['AFR', 'CPA', 'EEU', 'FSU', 'LAM']
+r2_northregs = ['EEU', 'FSU']
+r2_otherregs = ['AFR', 'CPA', 'LAM']
+
+WAVG_DF = pd.DataFrame([
+    ['model_a', 'scen_a', 'region_a', 'Primary Energy', 'EJ/y', 1, 6.],
+    ['model_a', 'scen_a', 'region_b', 'Primary Energy', 'EJ/y', 2, 12.],
+    ['model_a', 'scen_a', 'region_a', 'Price', 'EJ/y', 10, 5.],
+    ['model_a', 'scen_a', 'region_b', 'Price', 'EJ/y', 4, 5.],
+], columns=['model', 'scenario', 'region', 'variable', 'unit', 2005, 2010])
+
+
+@pytest.fixture(scope="function")
+def wavg_df():
+    df = pyam.IamDataFrame(data=WAVG_DF)
+    yield df
+
+
+TEST2_DF = pd.DataFrame([
```

Comment:
not sure why we even need such a detailed test dataframe - all behaviours should be checkable with `WAVG_DF`.

If really necessary, please follow the naming convention of other test dataframes.